app/models/company_value.rb

```diff
@@ -1,3 +1,5 @@
 class CompanyValue < ApplicationRecord
   validates :title, presence: true, uniqueness: true
+
+  has_many :kudos, class_name: 'Kudo', dependent: :destroy
```

Comment:
specifying the class name in such cases is not necessary :)

```suggestion
  has_many :kudos, dependent: :destroy
```